src/transforms/serializer.js

```diff
@@ -1,9 +1,9 @@
 'use strict'
 
-const ProtoDef = require('protodef').ProtoDef
-const Serializer = require('protodef').Serializer
-const Parser = require('protodef').FullPacketParser
-const { ProtoDefCompiler } = require('protodef').Compiler
+const ProtoDef = require('@saiv46/protodef').ProtoDef
+const Serializer = require('@saiv46/protodef').Serializer
+const Parser = require('@saiv46/protodef').FullPacketParser
+const { ProtoDefCompiler } = require('@saiv46/protodef').Compiler
```

Comment:
To improve overall esthetics of the code I'd use destructing there.
```js
const { ProtoDef, ..., Compiler: ProtoDefCompiler } = require(...)
```

Reply:
I just replaced "protodef" with "@saiv46/protodef" on master branch.

Reply:
Yeah, I know but if you can fix some styling in code you change then it's welcomed :)